packages/components/date-picker-panel/src/date-picker-com/panel-date-range.vue

```diff
@@ -244,10 +244,7 @@
         </div>
         <div :class="[ppNs.e('content'), drpNs.e('content')]" class="is-right">
           <div
-            :class="[
-              drpNs.e('header'),
-              ppNs.is('disabled', !enableYearArrow || dateRangeDisabled),
-            ]"
+            :class="[drpNs.e('header'), ppNs.is('disabled', dateRangeDisabled)]"
```

Comment:
The class can be deleted completely it has in fact no effect at all.
```suggestion
            :class="drpNs.e('header')"
```

Reply:
Agreed, lets delete it then. I wasn't sure if there was a reason it was there.